creator-node/src/tracer.ts

```diff
@@ -243,6 +243,16 @@ export const tracing = {
   /**
    * records errors on the current trace and sets the span status to `ERROR`
    * @param {Error} error the error to record on the span
+   *
+   * Usage
+   * ```
+   * try {
+   *     someSketchyCode()
+   * } catch (e: any) {
+   *     tracing.recordException(e)
```

Comment:
so if we forget to do this nothing bad really happens?

Reply:
Nope, it's basically just a log

Reply:
<img width="631" alt="Screen Shot 2022-08-30 at 10 35 36" src="https://user-images.githubusercontent.com/17736112/187465491-89caf719-2be7-4368-b721-cf457d07ec15.png">

It looks like this on CloudTrace